iocore/net/I_NetVConnection.h

```diff
@@ -644,6 +647,103 @@ class NetVConnection : public VConnection
   NetVConnection(const NetVConnection &) = delete;
   NetVConnection &operator=(const NetVConnection &) = delete;
 
+  enum ProxyProtocolVersion_t {
+    PROXY_VERSION_UNDEFINED,
+    PROXY_V1,
```

Comment:
PROXY_ here seems a little ambiguous in this flat name space. Can we make the symbols e.g. PROXY_PROTO_VERSION_UNDEF PROXY_PROTO_VERSION_1 and PROXY_PROT_VERSION_2 ? Similar to your name spacing further below (in ProxyProtocolData_t).

Reply:
That, or make `ProxyProtocolVersion` an `enum class` and change the names to `UNDEFINED` and `V1`.